packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -565,6 +580,10 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
                 command: CommonCommands.COLLAPSE_ALL_PANELS.id,
                 keybinding: 'alt+shift+c',
             },
+            {
+                command: CommonCommands.TOGGLE_MAXIMIZED.id,
+                keybinding: 'ctrlcmd+m',
```

Comment:
Note: it cannot be `ctrlcmd`. Only `ctrl` is valid on OS X. Otherwise, you overrule macOS's minimize binding.

It is up to you what you pick, but I wrote Ctrl+M and you interpreted my comment as CtrlCmd+M

Reply:
> I wonder how Eclipse deals with it.

It is ctrl+m in Eclipse on macOS and not cmd+m.

Reply:
You are right! just checked it, will change

had to trust to internet less and check with latest Eclipse first :)

Reply:
switched to `ctrl+m`, unfortunately terminals have already bindings for it, so it does not work for them